src/inmanta/execute/scheduler.py

```diff
@@ -362,18 +399,15 @@ def run(self, compiler: "Compiler", statements: Sequence["Statement"], blocks: S
         if i == max_iterations:
             raise CompilerException(f"Could not complete model, max_iterations {max_iterations} reached.")
 
-        # now = time.time()
-        # print(now - prev)
-        # end evaluation loop
-        # self.dump_not_done()
-        # print(basequeue, waitqueue)
-        # dumpHangs()
-        # self.dump()
-        # rint(len(self.types["std::Entity"].get_all_instances()))
-
         excns: List[CompilerException] = []
         self.freeze_all(excns)
 
+        now = time.time()
+        LOGGER.info(
+            "Total compilation time %f)",
```

Comment:
This looks like a typo.
```suggestion
            "Total compilation time %f",
```